tests/conftest.py

```diff
@@ -35,5 +33,25 @@ def generate_house(db):
 @pytest.fixture
 def generate_expense(db, generate_house):
     return Expenses.create_expense(
-        house_name=generate_house, amount=EXPENSE_AMOUNT, date=EXPENSE_DATE, category=EXPENSE_CATEGORY
+        house_name=generate_house, amount=EXPENSE_AMOUNT, date=EXPENSE_DATE, category=EXPENSE_CATEGORY,
+        description=EXPENSE_DESCRIPTION
     )
+
+
+@pytest.fixture
+def request_factory():
+    return RequestFactory()
+
+
+@pytest.fixture
+def generate_get_request(request_factory):
+    get_request = request_factory.get('login/')
+    return get_request
+
```

Comment:
Just like Rony's PR, both request_factory() and generate_get_request() are unused throughout the project and can be deleted.

Reply:
She needs to rebase and it will be fixed

Reply:
what Rony said :)